### DISCUSSION TITLE
Runner Pods is in Pending state due to Privileged container is not allowed: dind, securityContext: {"privileged": true} in the Organization Cluster

### DISCUSSION THREAD

**Discussion by AkshaySuryawanshi8** [actions/actions-runner-controller#2851] [Questions]
Hi Team,
need your help with problem I am facing with Runner Pods are been in pending state due to below error described in runner pod.

Overview: So I am using latest update of actions runner controller (runner scale set). so while deploying the same setup in my organization cluster I am facing issue with "Privileged container is not allowed: dind, securityContext: {"privileged": true}".

![image](https://github.com/actions/actions-runner-controller/assets/93468008/200f0627-316c-4815-8ec2-f2751c623293)

I did change the value of securityContext: {"privileged": false} in the spec sections of the template runner Pod inside the values.yml file of [gha-runner-scale-set](https://github.com/actions/actions-runner-controller/tree/master/charts/gha-runner-scale-set) still it show me the error.
so due to this my runner are showing offline in github runner group.
Could you please help me to fix this issue.

Thank you!

**Comment by davidparks21** [comment]
I'm experiencing the same problem here, I've tried two different values to set `privileged` to `false`. As the error suggest and as the k8s documentation suggests. Both still yield the same error:

```
// Kubernetes config
process.executor = 'k8s'
k8s.namespace = 'braingeneers'
k8s.computeResourceType = 'Job'
k8s.securityContext.privileged = false
k8s.securityContext.allowPrivilegeEscalation = false
```

**Comment by davidparks21** [comment]
I see the following related discussions that seem to suggest that this should work, but I'm having a hard time following these threads, or at least these threads seem to suggest that my `nextflow.config` is correct.

- https://github.com/actions/actions-runner-controller/issues/792
- https://github.com/actions/actions-runner-controller/pull/1383

**Comment by davidparks21** [comment]
I've moved this question over to StackOverflow here: https://stackoverflow.com/questions/77062492/how-to-disable-privileged-container-execution-in-nextflows-kubernetes-executor
